### ISSUE TITLE
minikube/docker 

### ISSUE THREAD

**Issue by raboof** [apache/pekko-management#196]
it seems the minikube/docker setup for the integration test is flaky:
```
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
```

**Comment by mdedetrich** [comment]
Yes this has been known for a while, historically I have just manually re-run tests until they pass but we should fix this properly (ideally using testcontainers as this solves the entire testing against docker problem)
